From 92f19473b28196492cbfd8153f8d48f2297a562a Mon Sep 17 00:00:00 2001
From: He Sheng <hesheng@wxiat.com>
Date: Fri, 6 May 2022 09:57:27 +0800
Subject: [PATCH] sw64: kapi: remove unused header-y from Kbuild
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: 92f19473b28196492cbfd8153f8d48f2297a562a
Modified-by-SEL: No


Sunway inclusion
category: feature
bugzilla: https://gitee.com/openeuler/kernel/issues/I56OLG

--------------------------------

Signed-off-by: He Sheng <hesheng@wxiat.com>

Signed-off-by: Gu Zitao <guzitao@wxiat.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/sw_64/include/asm/Kbuild | 19 ++++---------------
 1 file changed, 4 insertions(+), 15 deletions(-)

diff --git a/arch/sw_64/include/asm/Kbuild b/arch/sw_64/include/asm/Kbuild
index e276ba366e68..a94a978edf3e 100644
--- a/arch/sw_64/include/asm/Kbuild
+++ b/arch/sw_64/include/asm/Kbuild
@@ -1,23 +1,12 @@
 # SPDX-License-Identifier: GPL-2.0
-header-y += compiler.h
-header-y += console.h
-header-y += fpu.h
-header-y += gentrap.h
-header-y += hmcall.h
-header-y += reg.h
-header-y += regdef.h
-header-y += sysinfo.h
-header-y += page.h
-header-y += elf.h
 
-generated-y += syscall_table.h
+generic-y += clkdev.h
 generic-y += export.h
 generic-y += kvm_types.h
-generic-y += rwsem.h
-
+generic-y += mcs_spinlock.h
 generic-y += qrwlock.h
 generic-y += qspinlock.h
-generic-y += mcs_spinlock.h
-generic-y += clkdev.h
+generic-y += rwsem.h
 generic-y += scatterlist.h
 generic-y += user.h
+generated-y += syscall_table.h
-- 
2.33.0

